### ISSUE TITLE
argument `--slice-time-ref`: invalid _slice_time_ref value: '0'

### ISSUE THREAD

**Issue by bmelo** [nipreps/fmriprep#2572]
Hi all, we are trying to run the new fMRIPrep 20.2.4 release to use the new flag `--slice-time-ref`, but we are getting the following errors:

`fmriprep: error: argument --slice-time-ref: invalid _slice_time_ref value: 'start'`
`fmriprep: error: argument --slice-time-ref: invalid _slice_time_ref value: '0'`

It seems that the argument is wrong, but accordingly to the code it seems fine. Are we doing something wrong?

### What version of fMRIPrep are you using?

20.2.4

### What kind of installation are you using? Containers (Singularity, Docker), or "bare-metal"?

Singularity

### What is the exact command-line you used?
```
fmriprep-20.2.4.sif --slice-time-ref 0 $BIDSDIR $OUTPUTDIR participant --participant_label $SUBJID
```
and
```
fmriprep-20.2.4.sif --slice-time-ref start $BIDSDIR $OUTPUTDIR participant --participant_label $SUBJID
```

### Have you checked that your inputs are BIDS valid?

Yes.

### Did fMRIPrep generate the visual report for this particular subject? If yes, could you share it?

No.

### Can you find some traces of the error reported in the visual report (at the bottom) or in *crashfiles*?
```
fmriprep: error: argument --slice-time-ref: invalid _slice_time_ref value: '0'
```

### Are you reusing previously computed results (e.g., FreeSurfer, Anatomical derivatives, work directory of previous run)?

No.

### fMRIPrep log
If you have access to the output logged by fMRIPrep, please make sure to attach it as a text file to this issue.

> It's not starting.

**Comment by effigies** [comment]
Thanks for this. I'm able to reproduce the issue in tests. Will see what's up.

**Comment by mgxd** [comment]
I think the problem is https://github.com/nipreps/fmriprep/blob/5fe2c33ca98efdcc9fcc1327e78034b2848f61e8/fmriprep/cli/parser.py#L70 needs to be converted to a `partial` to be used as an argparse type, since it requires the parser object.

**Comment by effigies** [comment]
Yup. I didn't realize that was our own hack. Submitting a PR now.

**Comment by effigies** [comment]
Closed by #2573.
